app/feature/feature-movingflow/src/main/kotlin/com/hedvig/android/feature/movingflow/ui/selectcontract/SelectContractDestination.kt

```diff
@@ -45,14 +45,15 @@ internal fun SelectContractDestination(
   viewModel: SelectContractViewModel,
   navigateUp: () -> Unit,
   exitFlow: () -> Unit,
-  onNavigateToNextStep: (moveIntentId: String) -> Unit,
+  onNavigateToNextStep: (moveIntentId: String, popUpDestination: Boolean) -> Unit,
```

Comment:
Can we make this name a bit more descriptive regarding what we're popping here which is extra?